warp-core/src/test/scala/com/workday/warp/collectors/DefaultMeasurementCollectionControllerSpec.scala

```diff
@@ -76,9 +76,9 @@ class DefaultMeasurementCollectionControllerSpec extends WarpJUnitSpec with Core
       // execution tag with no meta tags
       ExecutionTag("key3", "val3"),
 
-      // Ignored duplicate/erroneous execution and definition tags
+      // Not ignoring duplicate execution and definition tags
       ExecutionTag("key3", "val3"),
-      // these meta tags should fail to insert
+      // these meta tags should no longer fail to insert
```

Comment:
Not sure it makes sense to say `no longer` since the person reading this in the future may not have historical context. Perhaps it's fine to just say `These meta tags should successfully insert despite the duplicate tag value on the test definition.`